### PR TITLE
docs: add apm documentation

### PR DESCRIPTION
**Pull request by xiangwanpeng**
# Which issue does this PR close?

# Rationale for this change
Add docs for the module APM.

# What changes are included in this PR?
Add docs for the module APM.

# Are there any user-facing changes?

# How does this change test
I test mdbook's render result locally using 'mdbook serve'.

I test 'Github Pages' render result using my fork repository, and deploy pages to https://xiangwanpeng.github.io/holoinsight-docs/ .

### CODE REVIEW

File: docs/src/en/user-guide/apm/send-traces-to-holoinsight.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Using OTel SDK
+> To be added.
+
+## Using SkyWalking Agent
+### If your application is deployed in a container
+1. Edit the Dockerfile to download [SkyWalking Agent](https://skywalking.apache.org/downloads/#Agents) and configure
+```
+# Download and unzip the SkyWalking Agent
+RUN wget https://archive.apache.org/dist/skywalking/java-agent/8.11.0/apache-skywalking-java-agent-8.11.0.tgz
```
Comment (sw1136562366) [inline docs/src/en/user-guide/apm/send-traces-to-holoinsight.md]: update skywalking java agent 8.11.0 to  8.15.0

File: docs/src/en/user-guide/apm/send-traces-to-holoinsight.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+The HoloInsight collector receives trace data based on the [OpenTelemetry](https://opentelemetry.io/) specification, which means that you can use almost any open-source SDK (OTel, SkyWalking, etc.) to collect data and report it uniformly, depending on your software environment and programming language.
+
+# How to configure
+
+> The `YOUR_APPLICATION_NAME` mentioned below is used to identify the `application` (aka the `service`), which should be the same as the application name specified by `-a` when installing HoloInsight Agent.
+
+## Using OTel SDK
+> To be added.
+
+## Using SkyWalking Agent
```
Comment (sw1136562366) [inline docs/src/en/user-guide/apm/send-traces-to-holoinsight.md:15]: You can add the usage link provided by the skywalking community 
https://hub.docker.com/r/apache/skywalking-java-agent

File: docs/src/en/user-guide/apm/send-traces-to-holoinsight.md (hunk 1 above)
Comment (xiangwanpeng) [reply docs/src/en/user-guide/apm/send-traces-to-holoinsight.md]: fixed.

File: docs/src/en/user-guide/apm/send-traces-to-holoinsight.md (hunk 2 above)
Comment (xiangwanpeng) [reply docs/src/en/user-guide/apm/send-traces-to-holoinsight.md:15]: added.

**Review by sw1136562366 — APPROVED** [review]
LGTM

**Pull request merged**